code-challenges.js

```diff
@@ -10,6 +10,26 @@ var people = [
 ]
 // Expected output example: "Ford Prefect is a hitchhiker." "Zaphod Beeblebrox is a president of the galaxy." "Arthus Dent is a radio employee."
 
+//declare a variable that will take in the object
+const outputString = (object) => {
+  //establish a variable that will house our mapped object
```

Comment:
While the argument name is technically arbitrary, it is important to communicate the input and output of your function. This input is actually an array. Map is not an object method, so really you knew that "object" was actually and array.